src/Manifolds.jl

```diff
@@ -151,9 +151,11 @@ import Base:
     inv,
     isempty,
     length,
+    Matrix,
```

Comment:
This might be unnecessary if we now only use the convert approach?

Reply:
Yes, it seems so.